src/main/java/frc/robot/RaspberryPI.java

```diff
@@ -0,0 +1,82 @@
+package frc.robot;
+import edu.wpi.first.networktables.DoubleSubscriber;
+import edu.wpi.first.networktables.NetworkTable;
+import edu.wpi.first.networktables.NetworkTableInstance;
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+public class RaspberryPI {
+	private double fps = 0;
+	private NetworkTable table;
+
+	//FPS Calculation
+	private DoubleSubscriber fpsCounter;
+	private DoubleSubscriber cubeYawSubscriber;
+	private DoubleSubscriber cubeDistanceSubscriber;
+	private DoubleSubscriber coneYawSubscriber;
+	private DoubleSubscriber coneDistanceSubscriber;
+	private double lastValue = 0;
+	private double lastTime = 0;
+	private Timer timer = new Timer();
+
+	/**Updates the FPS each iteration of the robot.*/
+	public RaspberryPI() {
+		timer.start();
+		table = NetworkTableInstance.getDefault().getTable("datatable");
+		fpsCounter = table.getDoubleTopic("x").subscribe(-1);
```

Comment:
Let's put the topic names in constants so they're easy to find. Also the FPS counter needs a better topic name than "x".